doc/upstream_release_process.md

```diff
@@ -143,9 +148,12 @@ Example:
 $ ./lp-bugs-released cloud-init 21.3 1867532 1911680 1925395 1931392 1931577 1932048 1940233 1940235 1940839
 ```
 
-
 ## Upload to ubuntu/devel
-TODO: link to SRU documentation
+```bash
+$ git checkout upstream/ubuntu/devel -b ubuntu/devel
```

Comment:
This should be a capital -B to ensure you are reseting any local branch to upstream.
We probably also want to ensure a git fetch upstream too to make sure latest is pulled locally too.
```suggestion
$ git fetch upstream
$ git checkout upstream/ubuntu/devel -B ubuntu/devel
```

Reply:
This section also might want to cross-reference https://github.com/canonical/uss-tableflip/blob/120c16055ed94b2a2a58390794ad5717340271d8/doc/ubuntu_release_process.md#upstream-snapshot-process